src/main/scala/io/iohk/ethereum/db/storage/KeyValueStorage.scala

```diff
@@ -62,6 +62,7 @@ private[storage] trait KeyValueStorage[K, V] {
 object Namespaces {
   val ReceiptsNamespace: IndexedSeq[Byte] = IndexedSeq[Byte]('r'.toByte)
   val HeaderNamespace: IndexedSeq[Byte] = IndexedSeq[Byte]('h'.toByte)
+  val HeaderNumberNamespace: IndexedSeq[Byte] = IndexedSeq[Byte]('n'.toByte)
```

Comment:
there should be another one already defined for this, `HeightsNamespace` see https://github.com/input-output-hk/etc-client/blob/phase/3/blockSync/src/main/scala/io/iohk/ethereum/db/storage/KeyValueStorage.scala#L68